service/src/main/java/bio/terra/cbas/common/exceptions/WorkflowOutputNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.cbas.common.exceptions;
+
+public class WorkflowOutputNotFoundException extends Exception {
```

Comment:
I think this is fine for now. If we end up with lots of individual exceptions for every possible error case, we might want to consolidate under a "workflow processing exception" with individual error messages, or have these all extend from a common super class. But as I said, I think it's fine for now.